app/services/account_reset/grant_requests_and_send_emails.rb

```diff
@@ -48,5 +49,17 @@ def grant_request_and_send_email(arr)
       end
       true
     end
+
+    def fraud_wait_period_not_met?(arr, now)
+      if (arr.user.fraud_review_pending? ||
+        arr.user.fraud_rejection?) &&
+         (fraud_wait_period_days > 0)
+        return arr.requested_at > (now - fraud_wait_period_days)
+      end
+    end
+
+    def fraud_wait_period_days
+      IdentityConfig.store.account_reset_fraud_user_wait_period_days.days
```

Comment:
I expect this will raise an error if it's configured to `nil`:

```
$ rails c
Loading development environment (Rails 7.1.3.2)
[1] pry(main)> nil.days
NoMethodError: undefined method `days' for nil
```

(If build doesn't fail, can we also ensure test coverage)